core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1478,10 +1488,37 @@ public ServerBuilder contentPreview(int length) {
         return contentPreview(length, ArmeriaHttpUtil.HTTP_DEFAULT_CONTENT_CHARSET);
     }
 
+    /**
+     * Sets the {@link RequestConverterFunction}s, {@link ResponseConverterFunction}
+     * and {@link ExceptionHandlerFunction}s for creating an {@link AnnotatedHttpServiceExtensions}.
+     *
+     * @param requestConverterFunctions the {@link RequestConverterFunction}s
+     * @param responseConverterFunctions the {@link ResponseConverterFunction}s
+     * @param exceptionHandlerFunctions the {@link ExceptionHandlerFunction}s
+     */
+    public ServerBuilder annotatedHttpServiceExtensions(
+            Iterable<? extends RequestConverterFunction> requestConverterFunctions,
+            Iterable<? extends ResponseConverterFunction> responseConverterFunctions,
+            Iterable<? extends ExceptionHandlerFunction> exceptionHandlerFunctions) {
```

Comment:
nit: Could you match the order of arguments of this method and  related methods with `AnnotatedHttpServiceExtensions` constructor’s arguments. 

Either way is fine if it does not make breaking change.

Reply:
Ok!